packages/lambda-powertools-middleware-correlation-ids/event-sources/alb.js

```diff
@@ -0,0 +1,50 @@
+const CorrelationIds = require('@dazn/lambda-powertools-correlation-ids')
+const Log = require('@dazn/lambda-powertools-logger')
+const consts = require('../consts')
+
+function isMatch (event) {
+  return event.hasOwnProperty('httpMethod') && event.requestContext.hasOwnProperty('elb')
+}
+
+function captureCorrelationIds ({ requestContext, headers }, { awsRequestId }, sampleDebugLogRate) {
+  if (!headers) {
+    Log.warn(`Request ${awsRequestId} is missing headers`)
+    return
+  }
+
+  const albRequestId = awsRequestId
+  const correlationIds = { awsRequestId, albRequestId }
+  for (const header in headers) {
+    if (header.toLowerCase().startsWith('x-correlation-')) {
+      correlationIds[header] = headers[header]
+    }
+  }
+
+  if (!correlationIds[consts.X_CORRELATION_ID]) {
+    correlationIds[consts.X_CORRELATION_ID] = albRequestId || awsRequestId
```

Comment:
`albRequestId` is always gonna equal `awsRequestId` here, right? if so, let's remove `albRequestId` here

Reply:
yep, removed